roles/overpass-api/tasks/main.yml

```diff
@@ -65,4 +72,3 @@
 - include: compile.yml
 - include: set_up_database.yml
 - include: web.yml
-- include: munin.yml
```

Comment:
Is there a reason to remove munin plugin ?

Reply:
No reason, just a mistake :)